config/initializers/sidekiq.rb

```diff
@@ -1,15 +1,28 @@
+return unless defined?(Sidekiq)
+
+# Disable SSL certificate verification if using Heroku Redis
```

Comment:
Heroku Redis is a common deployment option, and requires special connection config.